tests/test_hypersphere.py

```diff
@@ -7,7 +7,7 @@
 from geomstats.geometry.hypersphere import Hypersphere
 from geomstats.geometry.matrices import Matrices
 
-MEAN_ESTIMATION_TOL = 1e-6
+MEAN_ESTIMATION_TOL = 1e-3
```

Comment:
Why this change in tolerance?

Reply:
Because it doesn't work that well when changing the mean... But I guess this is expected from properties of an estimator, given that according to the central limit theorem, the sample mean has variance in sqrt(n), and n is 10^6 here.